packages/protocol/contracts/team/TimelockTokenPool.sol

```diff
@@ -19,6 +19,13 @@ import "lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 import "lib/openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
 import "../common/EssentialContract.sol";
 
+/// @title IDecimals
+/// @dev Serves for calculating the common denominator (TKO vs. strike price in stables)
+interface IDecimals {
```

Comment:
You don't have to define this interface even if you want to use the decimals() function, you can use IERC20 from OZ. But I don't think you should use IDecimals in the first place as we can assume TKO's decimals is always 18 on Both Taiko L1 and L2.

Reply:
`decimals()` is not part of `IER20.sol` but OK, i can remove and have it fixed unit.